assimilation_code/modules/utilities/netcdf_utilities_mod.f90

```diff
@@ -128,34 +131,43 @@ module netcdf_utilities_mod
 
 interface nc_put_variable
    module procedure nc_put_char_1d
-   module procedure nc_put_single_int_1d
-   module procedure nc_put_int_1d
-   module procedure nc_put_single_real_1d
-   module procedure nc_put_real_1d
    module procedure nc_put_char_2d
+   module procedure nc_put_int_0d
+   module procedure nc_put_int_1d
    module procedure nc_put_int_2d
-   module procedure nc_put_real_2d
    module procedure nc_put_int_3d
-   module procedure nc_put_real_3d
    module procedure nc_put_int_4d
+   module procedure nc_put_real_0d
+   module procedure nc_put_real_1d
+   module procedure nc_put_real_2d
+   module procedure nc_put_real_3d
    module procedure nc_put_real_4d
+   module procedure nc_put_double_0d
+   module procedure nc_put_double_1d
+   module procedure nc_put_double_2d
+   module procedure nc_put_double_3d
+   module procedure nc_put_double_4d
```

Comment:
needed 32-bit and 64-bit versions.  the question is why these are 'real' and 'double' when the others are 'float' and 'double'

Reply:
random fluctuations in the quantum continuum?  :)

Reply:
That wouldve been my guess ... just didn't want to seem ... _**obvious**_ ...